### PR TITLE
ML validation set is turned off when a fixed model is used

### PR DESCRIPTION
**Pull request by koenderks**
Implements https://github.com/jasp-stats/jasp-test-release/issues/274

### CODE REVIEW

**Comment by koenderks** [comment]
@timKDJ I think it is good to merge this pull request first before we start working on any more issues in the ml module

**Comment by AlexanderLyNL** [comment]
@koenderks could you please resolve the conflicts?

**Comment by koenderks** [comment]
Yes, give me half an hour then its done. I'm in the train

File: JASP-Engine/JASP/R/commonMachineLearningClassification.R [hunk 1]
```diff
@@ -136,16 +137,21 @@
   
   }
   
+  # Add common columns
   classificationTable$addColumnInfo(name = 'ntrain', title = 'n(Train)', type = 'integer')
-  classificationTable$addColumnInfo(name = 'nvalid', title = 'n(Validation)', type = 'integer')
+  if(options[["modelOpt"]] != "optimizationManual")
```
Comment (vandenman) [inline JASP-Engine/JASP/R/commonMachineLearningClassification.R]: there are two identical if statements, maybe
```r
if(options[["modelOpt"]] != "optimizationManual") {
  ...
}
```

Comment (koenderks) [reply JASP-Engine/JASP/R/commonMachineLearningClassification.R]: The order of the columns is set by this part right? All set sizes should be placed together, and all accuracy scores as well. That's why I did this, or can it be done in some other way?

Comment (JorisGoosen) [reply JASP-Engine/JASP/R/commonMachineLearningClassification.R]: @koenderks 's solution seems the right one here.

**Pull request merged**